app/playlist-daily/index/route.js

```diff
@@ -9,7 +9,7 @@ export default Route.extend({
       let day   = moment().format('DD');
       this.replaceWith('playlist-daily.date', year, month, day, {
         queryParams: {
-          scheduleStation: queryParams.scheduleStation || 'wqxr'
+          scheduleStation: queryParams.scheduleStation || 'jonathan-channel'
```

Comment:
What's happening here?

Reply:
whoops i did that while testing and forgot to take it out!